# These parameters are supposed to be loade with the Track class
# Standard format
stay_in:
    - [13.118556457465647, 13.573559543147326]
    - [11.618556435113906, 14.623559558793545]
    - [10.318556415742396, 12.823559531971455]
    - [10.168556413507222, 12.823559531971455]
    - [6.6185563606081, 15.123559566244126]
    - [8.068556382214783, 16.973559593811274]
    - [6.518556359117984, 17.87355960722232]
    - [5.0685563375113, 15.2735595684793]
    - [1.1685562793967712, 9.423559481307507]
    - [-0.8814437511506092, 6.473559437349081]
    - [-2.8814437809529316, 3.4735593926455977]
    - [-7.131443844282867, -2.4764406960163114]
    - [-9.231443875575305, -4.476440725818634]
    - [-12.631443926239253, -9.776440804794788]
    - [-10.881443900162221, -10.77644081969595]
    - [-9.681443882280828, -9.126440795109033]
    - [-7.531443850243331, -5.8264407459352014]
    - [-6.331443832361938, -6.076440749660492]
    - [-5.131443814480544, -6.776440760091305]
    - [-3.931443796599151, -8.17644078095293]
    - [-6.681443837577344, -12.476440845027923]
    - [-5.181443815225602, -13.426440859184027]
    - [-3.7314437936189186, -10.976440822676182]
    - [-2.7314437787177575, -9.326440798089266]
    - [-2.0314437682869446, -7.826440775737524]
    - [-0.631443747425319, -5.776440745190143]
    - [0.5685562704560745, -4.376440724328518]
    - [2.2185562950429905, -1.9764406885657309]
    - [2.818556303983687, -0.7264406699392794]
    - [3.5185563144145, 0.22355934421682377]
    - [4.7685563330409515, 2.3235593755092623]
    - [6.568556359863042, 4.573559409036875]
    - [7.168556368803738, 5.923559429153443]
    - [7.968556380724667, 6.873559443309546]
    - [9.018556396370887, 8.223559463426113]
    - [10.818556423192977, 10.87355950291419]
    - [12.668556450760125, 12.873559532716513]]
    - [[-5.831443824911357, -3.3764407094273565]
    - [-3.8314437951090348, -0.426440665468931]
    - [-2.331443772757293, 1.7735593673136236]
    - [3.1685563091990936, 9.87355948801303]
    - [5.968556350922345, 13.773559546127558]
    - [6.018556351667403, 13.823559546872616]
    - [9.318556400841235, 11.473559511854887]
    - [6.668556361353158, 7.5235594529953005]
    - [5.0685563375113, 5.423559421702862]
    - [-0.28144374220991253, -2.6764406989965437]
    - [-2.331443772757293, -5.726440744445085]

keep_out:
  - [13.118556457465647, 13.573559543147326]
  - [11.618556435113906, 14.623559558793545]
  - [10.318556415742396, 12.823559531971455]
  - [10.168556413507222, 12.823559531971455]
  - [6.6185563606081, 15.123559566244126]
  - [8.068556382214783, 16.973559593811274]
  - [6.518556359117984, 17.87355960722232]
  - [5.0685563375113, 15.2735595684793]
  - [1.1685562793967712, 9.423559481307507]
  - [-0.8814437511506092, 6.473559437349081]
  - [-2.8814437809529316, 3.4735593926455977]
  - [-7.131443844282867, -2.4764406960163114]
  - [-9.231443875575305, -4.476440725818634]
  - [-12.631443926239253, -9.776440804794788]
  - [-10.881443900162221, -10.77644081969595]
  - [-9.681443882280828, -9.126440795109033]
  - [-7.531443850243331, -5.8264407459352014]
  - [-6.331443832361938, -6.076440749660492]
  - [-5.131443814480544, -6.776440760091305]
  - [-3.931443796599151, -8.17644078095293]
  - [-6.681443837577344, -12.476440845027923]
  - [-5.181443815225602, -13.426440859184027]
  - [-3.7314437936189186, -10.976440822676182]
  - [-2.7314437787177575, -9.326440798089266]
  - [-2.0314437682869446, -7.826440775737524]
  - [-0.631443747425319, -5.776440745190143]
  - [0.5685562704560745, -4.376440724328518]
  - [2.2185562950429905, -1.9764406885657309]
  - [2.818556303983687, -0.7264406699392794]
  - [3.5185563144145, 0.22355934421682377]
  - [4.7685563330409515, 2.3235593755092623]
  - [6.568556359863042, 4.573559409036875]
  - [7.168556368803738, 5.923559429153443]
  - [7.968556380724667, 6.873559443309546]
  - [9.018556396370887, 8.223559463426113]
  - [10.818556423192977, 10.87355950291419]
  - [12.668556450760125, 12.873559532716513]
